etc/ui-config/mh_default_org/paella/config.json

```diff
@@ -119,6 +119,9 @@
     },
     "//**** For a list of available plugins and configuration, go to": "https://github.com/polimediaupv/paella/blob/master/doc/plugins.md",
     "list": {
+      "es.upv.paella.opencast.loader": {
+        "flavor": "*/*"
```

Comment:
Maybe it's better to have a list of accepted flavors

Reply:
done.